gui/preload.js

```diff
@@ -268,6 +268,7 @@ class IPC {
   message (...args) { return electron.ipcRenderer.invoke('message', ...args) }
   checkpoint (...args) { return electron.ipcRenderer.invoke('checkpoint', ...args) }
   versions (...args) { return electron.ipcRenderer.invoke('versions', ...args) }
+  data (...args) { return electron.ipcRenderer.invoke('data', ...args) }
```

Comment:
this is not needed here, this adds a method to the electron ipc client, but this ipc is not the sidecar ipc